openwisp_monitoring/check/tests/test_models.py

```diff
@@ -16,6 +17,11 @@ def test_check_str(self):
         c = Check(name='Test check')
         self.assertEqual(str(c), c.name)
 
+    def test_check_no_content_type(self):
```

Comment:
What are you testing here?

Reply:
This happens to be a rare case when you create a check without passing any content_object (currently no such check classes exist). The test is verifying that everything works fine in that case as a metric was created by passing `Check` as the **content_object** and yes it covers 2 uncovered lines :sweat_smile: